indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -229,7 +229,8 @@ def _send_emails(self, form):
                                    attachments=attachments)
             signals.core.before_notification_send.send('registration-custom-email', email=email,
                                                        registration=registration, form=form)
-            send_email(email, self.event, 'Registration', log_metadata={'registration_id': registration.id})
+            send_email(email, self.event, 'Registration', user=session.user,
+                       log_metadata={'registration_id': registration.id})
```

Comment:
You can use `registration.log(...)` which automatically passes the `log_metadata`

Reply:
but this is `send_email` which uses `_log_email` that uses `event.log(...)`

Reply:
oh right, nevermind